lib/utils.js

```diff
@@ -212,3 +212,9 @@ export const flattenObjectDeep = obj =>
     (acc, k) => (typeof obj[k] === 'object' ? { ...acc, ...flattenObjectDeep(obj[k]) } : { ...acc, [k]: obj[k] }),
     {},
   );
+
+export const stripHTML = HTMLContent => {
+  const regex = /(&nbsp;|<([^>]+)>)/gi;
```

Comment:
Let's rather use the `sanitize-html` package that's already installed in the frontend. It should work with something like:

```es6
return sanitizeHtml(content, { allowedTags: [], allowedAttributes: {} })
```